x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/dataframe/analyses/Classification.java

```diff
@@ -283,9 +283,7 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getVersion().onOrAfter(Version.V_7_10_0)) {
             out.writeNamedWriteableList(featureProcessors);
         }
-        if (out.getVersion().onOrAfter(Version.V_8_0_0)) {
-            out.writeBoolean(earlyStoppingEnabled);;
-        }
+        out.writeBoolean(earlyStoppingEnabled);;
```

Comment:
Please either add a version check here or remove the one from line 221 for consistency between read and write methods.

Reply:
Fixed in 68edfa91a00